bin/collector

```diff
@@ -1,6 +1,2 @@
 #!/usr/bin/env bash
-
-DEFAULT_TYPE=dev
-SERVICE_TYPE=${1:-$DEFAULT_TYPE}
-
-bin/sbt 'project zipkin-collector-service' "run -f zipkin-collector-service/config/collector-${SERVICE_TYPE}.scala"
+SERVICE_TYPE=${1:-dev} ./gradlew :zipkin-collector-service:run
```

Comment:
Hey there I was looking at this pull request with James and I had some comments. Everything looks to be working here, I just noticed the "dev" default set in multiple places. What can be done to improve this a bit is to use the gradle.properties file to set the `serviceType` default. That will eliminate the need to grab system environment variables. Also another thing that can be done is to use `-P` which sets a project property that overrides the default. So you could end up with something more like:
```./gradlew :zipkin-collector-service:run -PserviceType=${1}```
(in this case it may be needed to have a default defined here as well, ` -PserviceType=${1:-dev}` so as to not accidentally nullify the value, but I didn't test myself)

```
gradle --help
-P, --project-prop      Set project property for the build script (e.g. -Pmyprop=myvalue).
```